opt_maven_install.sh

```diff
@@ -36,6 +38,10 @@ else
 	  tar -xvf /tmp/hive-bin.tar.gz -C /tmp \
 	  && mv /tmp/apache-hive-${HIVE_VERSION}.redhat-${RH_HIVE_PATCH_VERSION}-bin/ \
 	  $HIVE_OUT
+	  #   remove log4j possibility to use JNDI
+	  pushd $HIVE_OUT
+      zip -q -d log4j-core-*.jar org/apache/logging/log4j/core/lookup/JndiLookup.class
+      popd
```

Comment:
Can we remove the need to perform pushd/popd operations by supplying the path to the log4j-core-related jobs explicitly? We may also need to point it to the lib directory too:

```bash
echo "Removing the log4j-core-related jars"
zip -q -d ${HIVE_OUT}/lib/log4j-core-*.jar org/apache/logging/log4j/core/lookup/JndiLookup.class

echo "Checking whether the jars were removed"
find ${HIVE_OUT} -name "log4j-core-*.jar"
```

Also: do we need to handle the edge case where zip fails and kills the container image build?

Reply:
>/tmp/opt_maven_install.sh: line 24: zip: command not found

Let's just pivot to using `find`?

Reply:
zip is decompressing too, added commit to add zip to yum installs on the CI portion of opt script, not sure if ART portion will need it...